nvtabular/inference/triton/__init__.py

```diff
@@ -250,6 +285,183 @@ def _generate_model_config(
+
+    nvt_step = model_config.ModelEnsembling.Step(model_name=name + "_nvt", model_version=-1)
+    for input_col in nvt_config.input:
+        nvt_step.input_map[input_col.name] = input_col.name
+    for output_col in nvt_config.output:
+        nvt_step.output_map[output_col.name] = output_col.name + "_nvt"
+
+    hugectr_step = model_config.ModelEnsembling.Step(model_name=name + "hctr", model_version=-1)
+    for input_col in hugectr_config.input:
+        hugectr_step.input_map[input_col.name] = input_col.name + "_nvt"
+    for output_col in hugectr_config.output:
+        hugectr_step.output_map[output_col.name] = output_col.name
+
+    config.ensemble_scheduling.step.append(nvt_step)
+    config.ensemble_scheduling.step.append(hugectr_step)
+
+    with open(os.path.join(output_path, "config.pbtxt"), "w") as o:
+        text_format.PrintMessage(config, o)
+    return config
+
+
+def _generate_tensorflow_config(model, name, output_path):
+    """given a workflow generates the trton modelconfig proto object describing the inputs
+    and outputs to that workflow"""
+    config = model_config.ModelConfig(
+        name=name, backend="tensorflow", platform="tensorflow_savedmodel"
+    )
+
+    for col in model.inputs:
+        config.input.append(
+            model_config.ModelInput(
+                name=col.name, data_type=_convert_dtype(col.dtype), dims=[-1, 1]
+            )
+        )
+
+    for col in model.outputs:
+        config.output.append(
+            model_config.ModelOutput(
+                name=col.name.split("/")[0], data_type=_convert_dtype(col.dtype), dims=[-1, 1]
+            )
+        )
+
+    with open(os.path.join(output_path, "config.pbtxt"), "w") as o:
+        text_format.PrintMessage(config, o)
+    return config
+
+
+def _generate_hugectr_config(name, output_path, hugectr_params, max_batch_size=None):
+    config = model_config.ModelConfig(name=name, backend="hugectr", max_batch_size=max_batch_size)
+
+    config.input.append(
+        model_config.ModelInput(name="DES", data_type=model_config.TYPE_FP32, dims=[-1])
+    )
+
+    config.input.append(
+        model_config.ModelInput(name="CATCOLUMN", data_type=model_config.TYPE_INT64, dims=[-1])
+    )
+
+    config.input.append(
+        model_config.ModelInput(name="ROWINDEX", data_type=model_config.TYPE_INT32, dims=[-1])
+    )
+
+    for i in range(hugectr_params["n_outputs"]):
+        config.output.append(
+            model_config.ModelOutput(
+                name="OUTPUT" + str(i), data_type=model_config.TYPE_FP32, dims=[-1]
+            )
+        )
+
+    config.instance_group.append(model_config.ModelInstanceGroup(gpus=[0], count=1, kind=1))
+
+    config_hugectr = model_config.ModelParameter(string_value=hugectr_params["config"])
+    config.parameters["config"].CopyFrom(config_hugectr)
+
+    gpucache_val = "true"
+    if "gpucache" in hugectr_params.keys():
+        gpucache_val = hugectr_params["gpucache"]
```

Comment:
Use the dict 'get' method here:
```suggestion
    gpucache_val = hugectr_params.get("gpucache", "true")
```